niworkflows/interfaces/registration.py

```diff
@@ -88,14 +89,40 @@ class ApplyTOPUPRPT(nrc.RegistrationRC, fsl.ApplyTOPUP):
     output_spec = ApplyTOPUPOutputSpecRPT
 
     def _post_run_hook(self, runtime):
-        self._fixed_image_label = "before"
-        self._moving_image_label = "after"
-        self._fixed_image = index_img(self.inputs.in_files[0], 0)
-        self._moving_image = index_img(self.aggregate_outputs().out_corrected, 0)
-        NIWORKFLOWS_LOG.info('Report - setting before (%s) and after (%s) images',
+        self._fixed_image_label = "corrected"
+        self._moving_image_label = "warped"
+        self._fixed_image = index_img(self.aggregate_outputs().out_corrected, 0)
+        self._moving_image = index_img(self.inputs.in_files[0], 0)
+        self._contour = self.inputs.wm_seg if isdefined(self.inputs.wm_seg) else None
```

Comment:
http://nipype.readthedocs.io/en/latest/interfaces/generated/nipype.interfaces.fsl.epi.html#applytopup

Does not look like it from the docs.